lib/drives/models/drive.dart

```diff
@@ -144,6 +149,11 @@ class Drive extends Trip {
     //the rides get updated automatically by a supabase function.
   }
 
+  int getUnreadMessagesCount() {
+    if (chats == null) return 0;
+    return chats!.fold(0, (int value, Chat chat) => value + chat.getUnreadMessagesCount());
```

Comment:
this is still wrong. it should show the number of chats with new messages, not the total number of messages

Reply:
I'm not sure. I thougth the total number would be better what do the others think?

Reply:
I think it's kind of convention for chat apps to show the number of chats with new messages at that level